app/assets/stylesheets/custom.scss

```diff
@@ -321,3 +321,25 @@ ul, menu, dir {
     width: 100px;
     height: 30px;
 }
+
+#flash_notice {
```

Comment:
Avoid using id selectors<br>Selector `flash_notice` should be written in lowercase with hyphens